docs/docs/glossary.md

```diff
@@ -0,0 +1,19 @@
+# Glossary
+
+- [Glossary](#glossary)
+  - [AVS (Actively Validated Service)](#avs-actively-validated-service)
+  - [AVS Node](#avs-node)
+  - [Strategy contract](#strategy-contract)
+
+
+## AVS (Actively Validated Service)
+
+Collection of services or middleware software that is built on top of EigenLayer.
+
+## AVS Node
+
+An AVS Node is a node that runs the AVS software. Can be defined as a collection of Docker services declared in a Docker Compose file.
```

Comment:
I would change this to express that an AVS Node is an abstract entity in the protocol description, whereas the eigenlayer *operator* is the actual person running the software.

Maybe "An AVS Node is a node that runs the AVS software as an Eigenlayer Operator."?

Reply:
Done [c9ed26e](https://github.com/NethermindEth/el-node-spec/pull/58/commits/c9ed26e9d2d3a7d95fb99b8dc0678e22cd688b5e)

Reply:
thanks. can resolve